brainfuck/brainfuck_test.go

```diff
@@ -0,0 +1,152 @@
+package brainfuck
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+)
+
+// Test of the interpreter
+func TestInterpret(t *testing.T) {
+	rescueStdout := os.Stdout
+	r, w, _ := os.Pipe()
+	os.Stdout = w
+
+	Interpret("+++++c>p++++++[-<+>]<.")
+
+	w.Close()
+	out, _ := ioutil.ReadAll(r)
+	os.Stdout = rescueStdout
+
+	if out[0] != 16 {
+		t.Errorf("The output of programm +++++c>p+++++[-<+>]<. should be 16, but it equals %d", out[0])
+	}
+}
+
+func TestExecute(t *testing.T) {
```

Comment:
it's test for command.go. Pls create `command_test.go` and move it there

Reply:
Done